src/main/java/Event.java

```diff
@@ -0,0 +1,13 @@
+public class Event extends Task{
+
+    protected String at;
+
+    public Event(String name, String at){
```

Comment:
Suggestion:
Some of these method could have some comments on @param and @return.